core/templates/dev/head/pages/exploration_player/PlayerServices.js

```diff
@@ -369,9 +389,16 @@ oppia.factory('oppiaPlayerService', [
           exploration.isInteractionInline(oldStateName));
 
         if (!_editorPreviewMode) {
+          var isFirstHit = false;
+          if (visitedStates.indexOf(newStateName) == -1) {
+            isFirstHit = true;
+          }
           StatsReportingService.recordStateTransition(
             oldStateName, newStateName, answer,
-            LearnerParamsService.getAllParams());
+            LearnerParamsService.getAllParams(), isFirstHit);
+          visitedStates.push(newStateName);
+          _checkTimeSinceStart(
```

Comment:
Why are we recording "I've really started the exploration" every time a state is answered?

Reply:
@seanlip Thanks for the catch. Gotta set a Boolean field ExplorationActuallyStarted to True after recording it the first time. Missed that.